package.json

```diff
@@ -49,7 +49,7 @@
     "test:ut:integration:ci": "npm run test:ut:integration -- --expand --maxWorkers=50%",
     "pre-commit": "npm run test:ts:silent && npm run test:js:silent && npx lint-staged",
     "commit-msg": "commitlint --edit",
-    "prepare": "node ./scripts/skipPrepareScript.js || husky install",
+    "prepare": "npm --prefix ./node_modules/isolated-vm run rebuild",
```

Comment:
Left the `prepare` as is, in case we want to add preparation scripts for more packages in the future